Please update project to the latest renderers version

I'm getting a crash in such configuration and multidex on.
```
debugCompile 'com.github.pedrovgs:lynx:1.6'
compile 'com.github.pedrovgs:renderers:3.2.0'
```
```
Caused by: java.lang.NoSuchMethodError: No direct method <init>(Landroid/view/LayoutInflater;Lcom/pedrogomez/renderers/RendererBuilder;Lcom/pedrogomez/renderers/AdapteeCollection;)V in class Lcom/pedrogomez/renderers/RendererAdapter; or its super classes (declaration of 'com.pedrogomez.renderers.RendererAdapter' appears in /data/app/com.lalafo.dev-1/base.apk:classes48.dex)
                                                                    at com.github.pedrovgs.lynx.LynxView.initializeRenderers(LynxView.java:279)
                                                                    at com.github.pedrovgs.lynx.LynxView.initializeView(LynxView.java:246)
                                                                    at com.github.pedrovgs.lynx.LynxView.<init>(LynxView.java:89)
                                                                    at com.github.pedrovgs.lynx.LynxView.<init>(LynxView.java:82)
                                                                    at java.lang.reflect.Constructor.newInstance0(Native Method) 
                                                                    at java.lang.reflect.Constructor.newInstance(Constructor.java:430) 
                                                                    at android.view.LayoutInflater.createView(LayoutInflater.java:645) 
                                                                    at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:787) 
                                                                    at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:727) 
                                                                    at android.view.LayoutInflater.inflate(LayoutInflater.java:495) 
                                                                    at android.view.LayoutInflater.inflate(LayoutInflater.java:426) 
                                                                    at android.view.LayoutInflater.inflate(LayoutInflater.java:377) 
                                                                    at com.android.internal.policy.PhoneWindow.setContentView(PhoneWindow.java:414) 
                                                                    at android.app.Activity.setContentView(Activity.java:2414) 
                                                                    at com.github.pedrovgs.lynx.LynxActivity.onCreate(LynxActivity.java:58) 
                                                                    at android.app.Activity.performCreate(Activity.java:6662) 
                                                                    at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1118) 
                                                                    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2599) 
                                                                    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2707) 
                                                                    at android.app.ActivityThread.-wrap12(ActivityThread.java) 
                                                                    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1460) 
                                                                    at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                    at android.os.Looper.loop(Looper.java:154) 
                                                                    at android.app.ActivityThread.main(ActivityThread.java:6077) 
                                                                    at java.lang.reflect.Method.invoke(Native Method) 
                                                                    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:865) 
                                                                    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755) 
```

--- a/lynx/build.gradle
+++ b/lynx/build.gradle
@@ -22,7 +22,7 @@ android {
 
 dependencies {
   compile fileTree(dir: 'libs', include: ['*.jar'])
-  compile 'com.github.pedrovgs:renderers:3.0.0'
+  compile 'com.github.pedrovgs:renderers:3.2.0'
   compile 'com.squareup:seismic:1.0.2'
   testCompile 'org.robolectric:robolectric:3.1'
   testCompile 'junit:junit:4.12'

